packages/utils/package.json

```diff
@@ -5,21 +5,23 @@
     "homepage": "https://github.com/trezor/trezor-suite/packages/utils",
     "description": "A collection of typescript utils that are intended to be used across trezor-suite monorepo.",
     "license": "SEE LICENSE IN LICENSE.md",
+    "private": true,
```

Comment:
this package is not private, it is used as not direct dependency of trezor-connect, via blockchain-link for example

Reply:
definitely not private.

Reply:
fixed